lib/appoptics-services/output.rb

```diff
@@ -102,25 +104,28 @@ def generate_alert_cleared
         lines.join("\n")
       end
 
-      def format_measurement(measurement, source = nil)
+      def format_measurement(measurement, source = nil, service = nil)
         condition = @conditions[measurement[:condition_violated]]
         if source
           metric = "`#{measurement[:metric]}` from `#{source}`"
         else
           metric = "`#{measurement[:metric]}`"
         end
         violation_time = measurement[:end] || measurement[:recorded_at]
-        "metric #{metric} was #{format_violation_type(condition, measurement)} recorded at #{format_time(violation_time)}"
+        "metric #{metric} was #{format_violation_type(condition, measurement, service)} recorded at #{format_time(violation_time)}"
       end
 
-      def format_violation_type(condition, measurement)
+      def format_violation_type(condition, measurement, service)
         if condition[:type] == "absent"
           "absent for #{condition[:duration]} seconds"
         else
           threshold_value = condition[:threshold]
           actual_value = measurement[:value]
           formatted_value = AppOptics::Services::Numbers.format_for_threshold(threshold_value, actual_value)
-          "#{condition[:type]} threshold #{threshold(condition,measurement)} with value #{formatted_value}"
+          if service == AppOptics::Services::Service::Slack::SERVICE_NAME
```

Comment:
Is there a reason that only Slack should benefit from this change?

Reply:
I can't find the conversation right now but it was pointed out that change in webhooks, for example, might break something for the customers so it was decided that we should only change Slack output.